modules/@apostrophecms/doc-type/index.js

```diff
@@ -129,10 +129,10 @@ module.exports = {
             });
           }
 
-          if (doc._parentUrl) {
+          if (doc._parentPage) {
```

Comment:
If you are only going to capture the slug in this property then please call it `_parentSlug`.